setup.py

```diff
@@ -15,7 +15,7 @@
     maintainer_email="aesara-devs@gmail.com",
     packages=["aemcmc"],
     install_requires=[
-        "numpy>=1.18.1",
+        "numpy>=1.21.0",
```

Comment:
Do we still need this restriction?

Reply:
The library does import `numpy` as well, so I'd think yes?